src/wallet/rpc/backup.cpp

```diff
@@ -1258,7 +1263,8 @@ RPCHelpMan importmulti()
             "may report that the imported keys, addresses or scripts exist but related transactions are still missing.\n"
             "The rescan parameter can be set to false if the key was never used to create transactions. If it is set to false,\n"
             "but the key was used to create transactions, rescanwallet needs to be called with the appropriate block range.\n"
-            "Note: Use \"getwalletinfo\" to query the scanning progress.\n",
+            "Note: Use \"getwalletinfo\" to query the scanning progress.\n"
+            "Note: This command is only compatible with legacy wallets. Use \"importdescriptors\" with \"addr(X)\" for descriptor wallets.\n",
```

Comment:
I don't think it's useful to mention `addr()` here as `importmulti` allows the import of a ton of different things, including descriptors. Mentioning a specific descriptor type, which only applies to one of the things that `importmulti` can deal with, doesn't seem to make sense to me.

Reply:
@achow101 I agree, I'd rather like it as **"Use "importdescriptors" for descriptor wallets."**, mentioning a specific parameter doesn't seem necessary. People would get help by using only **"importdescriptors"** anyway.

This line can also be changed:
https://github.com/bitcoin/bitcoin/blob/1abbae65eb9c99df5d8941008068d83ad99bf117/src/wallet/rpc/backup.cpp#L213